docker/nginx.json

```diff
@@ -45,6 +45,11 @@
         "pass": "applications/rdwatch"
       }
     },
+    {
+      "match": {
+        "uri": "/staticfiles/*"
```

Comment:
should this be /static/  The `STATIC_URL` in the settings.py is `static` the `STATIC_ROOT` which I think is the file location is `staticfiles`?